src/services/opensrp/index.ts

```diff
@@ -196,7 +196,7 @@ export class OpenSRPService {
    * @param {HTTPMethod} method - the HTTP method
    * @returns the object returned by API
    */
-  public async delete<T>(params: paramsType = null, method: HTTPMethod = 'DELETE') {
+  public async delete(params: paramsType = null, method: HTTPMethod = 'DELETE') {
```

Comment:
Can we move `"DELETE"` to `contsants.ts`?

Reply:
I think we should fix that through this issue https://github.com/onaio/reveal-frontend/issues/682